### ISSUE TITLE
Crash on the Insights tab in stats

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#9746]
The app crashes everytime I try to open the Stats on a specific site. Stack trace:

```
2019-04-29 14:56:42.337 4155-4155/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android, PID: 4155
    java.util.NoSuchElementException: List is empty.
        at kotlin.collections.CollectionsKt___CollectionsKt.last(_Collections.kt:360)
        at org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases.AnnualSiteStatsUseCase.buildUiModel(AnnualSiteStatsUseCase.kt:47)
        at org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases.AnnualSiteStatsUseCase.buildUiModel(AnnualSiteStatsUseCase.kt:20)
        at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$StatelessUseCase.buildUiModel(BaseStatsUseCase.kt:255)
        at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$StatelessUseCase.buildUiModel(BaseStatsUseCase.kt:240)
        at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$liveData$1.invoke(BaseStatsUseCase.kt:42)
        at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$liveData$1.invoke(BaseStatsUseCase.kt:29)
        at org.wordpress.android.util.LiveDataUtilsKt$merge$7.invoke(LiveDataUtils.kt:129)
        at org.wordpress.android.util.LiveDataUtilsKt$merge$7.invoke(Unknown Source:2)
        at org.wordpress.android.util.LiveDataUtilsKt$map$1.onChanged(LiveDataUtils.kt:161)
```

(Crash happened on e30f2c2abdc7009d8c96819f0e1567868f585f21)

**Comment by malinajirka** [comment]
I believe a fix has been recently merged https://github.com/wordpress-mobile/WordPress-Android/pull/9787. @planarvoid Can you please confirm it's the same issue. Thanks!

**Comment by malinajirka** [comment]
I've just checked it and it seems like the same issue. 

Closing - fixed in #9787
